frontend/components/common/formatters/AnimatedNumber.vue

```diff
@@ -34,11 +34,17 @@ export default Vue.extend({
         },
     },
     computed: {
+        parsedValue(): number | BigNumber {
+            if (typeof this.value === 'string') {
+                return parseInt(this.value);
+            }
+            return this.value;
+        },
```

Comment:
Why is there changes to the AnimatedNumber in this PR? Seems unrelated to me (and again converts BigNumber into number which we agreed to avoid in the past)